TileStache/Goodies/Providers/Composite.py

```diff
@@ -451,13 +451,13 @@ def make_color(color):
 def _arr2img(ar):
     """ Convert Numeric array to PIL Image.
     """
-    return Image.fromstring('L', (ar.shape[1], ar.shape[0]), ar.astype(numpy.ubyte).tostring())
+    return Image.frombytes('L', (ar.shape[1], ar.shape[0]), ar.astype(numpy.ubyte).tobytes())
 
 def _img2arr(im):
     """ Convert PIL Image to Numeric array.
     """
     assert im.mode == 'L'
-    return numpy.reshape(numpy.fromstring(im.tostring(), numpy.ubyte), (im.size[1], im.size[0]))
+    return numpy.reshape(numpy.frombytes(im.tobytes(), numpy.ubyte), (im.size[1], im.size[0]))
```

Comment:
This seems to be wrong http://docs.scipy.org/doc/numpy/reference/generated/numpy.fromstring.html